analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/Verifier.cs

```diff
@@ -199,8 +226,26 @@ private IEnumerable<Compilation> CompileRazor()
                     Directory.Delete(tempPath, true);
                 }
             }
+
+            IEnumerable<ParseOptions> GetParseOptions() =>
```

Comment:
Instead of this method I suggest to use the language version `latest` if there is nothing else defined.